include/GafferOSL/OSLCode.h

```diff
@@ -72,6 +72,8 @@ class OSLCode : public OSLShader
 		/// combo we use everywhere else.
 		ShaderCompiledSignal &shaderCompiledSignal();
 
+		virtual void loadShader( const std::string &shaderName, bool keepExistingValues=false );
```

Comment:
It depends if we want the base class method to be pure virtual or not I guess. Seems like it should be in an ideal world, but OSLCode and ArnoldDisplacement spoil things a bit by not quite fitting the mould. I'll leave it up to you, but I think I'd lean towards making it pure virtual as a guide to the implementers of future subclasses, and so that we're forced to make an explanatory comment about OSLCode...